### ISSUE TITLE
don't require namespace to be provided

### ISSUE THREAD

**Issue by trisberg** [cppforlife/knctl#10]
would be more convenient if knctl used the default namespace defined in kubeconfig, there is no need to provide namespace for kubectl

**Comment by cppforlife** [comment]
oh, i did not know that...

```
$ kubectl config set-context $(kubectl config current-context) --namespace=<insert-namespace-name-here>
# Validate it
$ kubectl config view | grep namespace:
```

from https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/

**Comment by cppforlife** [comment]
will add.

**Comment by cppforlife** [comment]
fixed.

**Comment by cppforlife** [comment]
included in v0.0.5
